client/my-sites/site-settings/form-general.jsx

```diff
@@ -475,6 +475,37 @@ class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	dateTimeFormat() {
+		if ( ! config.isEnabled( 'manage/site-settings/date-time-format' ) ) {
+			return null;
+		}
+
+		const {
+			fields: {
+				date_format,
+				start_of_week,
+				time_format,
+			},
+			site,
+			translate,
+		} = this.props;
+
+		// Placeholder formats
+		return (
+			<Card
+				className="site-settings__date-time-format"
+				href={ `/settings/date-time-format/${ site.slug }` }
+			>
+				<h2 className="site-settings__date-time-format-title">
+					{ translate( 'Date and time format configuration' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **52** times:
`translate( 'Date and time format:' )` *ES Score: 6.74*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-settings/date-format-sub-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).